shared/templates/grub2_bootloader_argument/blueprint.template

```diff
@@ -0,0 +1,4 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ubuntu,multi_platform_sle
```

Comment:
Any reason for not putting just multi_platform_all here?
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#header

Reply:
Parts of BP won't be available for all platforms. It is to be refined further, but the current one I just copied from the sibling template.